apps/frontend/src/app/assets/constants/Config/DashboardConfig/index.js

```diff
@@ -0,0 +1,30 @@
+import { piedata } from "app/assets/constants/mockData";
```

Comment:
@Nazi-pikachu please take the constants and styles folder out of it

Reply:
resolved